tests/area/test_markets.py

```diff
@@ -74,8 +78,10 @@ def test_market_rotation_is_successful(self, area_fixture):
         assert len(area_fixture.past_markets) == 1
         assert len(area_fixture.all_markets) == 1
 
+    @staticmethod
     @patch("gsy_e.constants.RETAIN_PAST_MARKET_STRATEGIES_STATE", True)
-    def test_market_rotation_is_successful_keep_past_markets(self, area_fixture):
+    def test_market_rotation_is_successful_keep_past_markets(area_fixture):
+        GlobalConfig.FUTURE_MARKET_DURATION_HOURS = 24
```

Comment:
This is the actual fix, this test depended on the future market hours having a value of 24.

Reply:
Shouldn't we restore the original value here as well? I know that `24` is the default, but if we change the default value this might create issues, right?